File: android/app/src/main/java/org/pytgcalls/ntgcalls/media/FrameData.java
package org.pytgcalls.ntgcalls.media;

public class FrameData {
    public final long absoluteCaptureTimestampMs;
    public final int width, height, rotation;

    public FrameData(long absoluteCaptureTimestampMs, int width, int height, int rotation) {
        this.absoluteCaptureTimestampMs = absoluteCaptureTimestampMs;
        this.width = width;
        this.height = height;
        this.rotation = rotation;
    }
}

File: wrtc/src/models/frame_data.cpp
//
// Created by Lauren on 20/02/25.
//

#include <wrtc/models/frame_data.hpp>

namespace wrtc::models {

    FrameData::FrameData(
        const int64_t absolute_capture_timestamp_ms,
        const webrtc::VideoRotation rotation,
        const uint16_t width,
        const uint16_t height
    ): absolute_capture_timestamp_ms(absolute_capture_timestamp_ms), rotation(rotation), width(width), height(height) {}

} // wrtc::models
